DEPATISnet.js

```diff
@@ -197,13 +197,14 @@ function scrape(doc, url) {
 		snapshot: false
 	});
 	
-	var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
+	var pages = ZU.xpathText(doc, '/html/body/div[4]/p/span');			// old line: var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
```

Comment:
This is too fragile. It looks like you can perhaps just get this from the `caption` in the `table`, but if that's not on every page, you'd want to use at least the class name of the containing `p`.

For new code, you can also just use `text()` and CSS selectors instead of XPaths. Example: https://github.com/zotero/translators/blob/1d1ce6111ac17a4064c1d8836f6beec54404c32b/Stitcher.js#L83

Reply:
I have changed this line to

`var pages = ZU.xpathText(doc, '//table/caption').replace("(", "$(").split("$").pop();`

I hope thats better.